src/cache-utils.js

```diff
@@ -147,6 +147,7 @@ function cacheTypeForPayload (payload) {
     case 'shh_getFilterChanges':
     case 'shh_getMessages':
     case 'test_neverCache':
+    default:
       return 'never'
   }
```

Comment:
This switch case for cache method had `default-case` error. Since there are multiple cases are using `never` cache, I made an assumption that the `default` condition will be `never`. Is there any other value that can be defined?